mod/messageboard/start.php

```diff
@@ -114,7 +114,11 @@ function messageboard_add($poster, $owner, $message, $access_id = ACCESS_PUBLIC)
 			$poster->getURL()
 		), $owner->language);
 
-		notify_user($owner->guid, $poster->guid, $subject, $body);
+		$params = [
+			'action' => 'create',
```

Comment:
I will be fine with matching with the action name. I don't see any downsides since we don't really trigger granular hooks on these.